burn-core/Cargo.toml

```diff
@@ -60,6 +60,7 @@ blas-netlib = ["burn-ndarray?/blas-netlib"]
 
 ndarray = ["burn-ndarray"]
 tch = ["burn-tch"]
+tch-doc-only = ["burn-tch/doc"]
```

Comment:
I would rename the feature flag to `doc-only` for all crates (burn-tch, burn-core and burn)